sdgx/data_models/relationship.py

```diff
@@ -0,0 +1,26 @@
+from pydantic import BaseModel
+
+
+class Relationship(BaseModel):
+    """Relationship between tables
+
+    For parent table, we don't need define primary key here.
+    The primary key is pre-defined in parent table's metadata.
+
+    Child table's foreign key should be defined here.
+    """
+
+    metadata_version: str = "1.0"
+
+    # table names
+    parent_table: str
+    child_table: str
+
+    # keys
+    child_table_foreign_key: str
```

Comment:
seems should be a list, as primary key can be composite

```suggestion
    child_table_foreign_key: List[str]
```